config/templates/cluster.yaml

```diff
@@ -121,6 +130,18 @@ worker:
 # (Set to an odd number >= 3 for HA control plane)
 # etcdCount: 1
 
+# etcd:
+#   privateSubnets:
+#     - availabilityZone: us-west-1a
+#       instanceCIDR: "10.0.2.0/24"
+#     - availabilityZone: us-west-1b
+#       instanceCIDR: "10.0.3.0/24"
+#       # natGateway:
+#       #   # Pre-allocated NAT Gateway. Used with private subnets.
+#       #   id: "ngw-abcdef12"
+#       #   # Pre-allocated EIP for NAT Gateways. Used with private subnets.
+#       #   eipAllocationIDs: "eipalloc-abcdef12"
```

Comment:
Oh, I've noticed that the corresponding model is already made so 👍 https://github.com/neoandroid/kube-aws/pull/1/files#diff-c76375ae6854b059e3f028487955882dR14